src/Execute/ExecutionEnv.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Command} from '../Backend/Command';
+
+interface ExecutionEnv {
+  name(): string;
+  envInfo(): string;
+  runCommand(command: Command): Command;
+}
+
+// This is a base ExeuctionEnv.
+// on New ExecutionEnv implements, need to add 3 things
```

Comment:
```suggestion
// on new ExecutionEnv implements, need to add 3 things
```

Reply:
Does _new ExecutionEnv_ mean a class that inherits `ExecutionEnvBase`?

Reply:
👍 I will fix it. thanks.